website/models.py

```diff
@@ -36,3 +36,19 @@ def instance():
             return Settings.objects.get()
         except Settings.DoesNotExist:
             return None
+
+class Schedule(models.Model):
+    """A Schedule for a Degree during the Ophase"""
+    class Meta:
+        verbose_name = _("Stundenplan")
+        verbose_name_plural = _("Stundenpläne")
+
+    DEGREE_CHOICES = (
+        ('BSC', _('Bachelor')),
+        ('MSC', _('Master Deutsch')),
+        ('DSS', _('Distributed Software Systems')),
+    )
+
+    degree = models.CharField(max_length=3, choices=DEGREE_CHOICES, verbose_name=_('Abschluss'), unique=True)
+    image = models.ImageField(upload_to='website/schedule/', verbose_name=_('Stundenplan Bild'))
```

Comment:
Using the upload this way leads to a cluttered upload folder containing multiple old versions of the schedule after a time. I would prefer giving the file a fixed name and use `storage=OverwriteStorage()` attribute of the `ImageField`.
Where `OverwriteStorage` is
```python
class OverwriteStorage(FileSystemStorage):
    def get_available_name(self, name):
        self.delete(name)
        return name
```
from http://stackoverflow.com/a/19961911

But it might be the case that it is tricky to give it a fixed name based on the degree field at the time of object creation. So if you don't like this, skip it for the moment.